src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -85,21 +87,49 @@ public void patchObjection(String requestId, String companyNumber, String object
     }
 
     @Override
-    public ServiceResult<String> addAttachment(String requestId, MultipartFile file) throws ServiceException {
-        FileTransferApiClientResponse response = fileTransferApiClient.upload(requestId, file);
+    public void addAttachment(String requestId, String objectionId, MultipartFile file, String attachmentsUri)
+            throws ServiceException, ObjectionNotFoundException {
 
+        FileTransferApiClientResponse response = fileTransferApiClient.upload(requestId, file);
```

Comment:
A couple of debug/info log statements either side of this call to another API might be useful for investigating any issues in the future.